### PR TITLE
Status update 2020-01-08 

### PR DESCRIPTION
**Pull request by pmcmaw**

### CODE REVIEW

File: _posts/2021-01-08-status-update.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  - rubocop
+  - infrastructure
+  - networking
+  - puppetize
+  - triage
+---
+
+## Happy New Year!
+
+On behalf whole IAC team we would like to take the opportunity to wish all readers a Happy New Year! We hope you all head a great Christmas. The team took the opportunity enjoy the holidays, recharge our batteries and are now ready to tackle 2021 head on.
```
Comment (sanfrancrisko) [inline _posts/2021-01-08-status-update.md:24]:
```suggestion
On behalf whole IAC team we would like to take the opportunity to wish all readers a Happy New Year! We hope you all had a great Christmas. The team took the opportunity enjoy the holidays, recharge our batteries and are now ready to tackle 2021 head on.
```

File: bin/blog_tools/blog_template/status-update-template.md.erb [hunk 2]
```diff
@@ -43,4 +43,5 @@ The following modules were released this week:
   [Paula]:              https://github.com/pmcmaw
   [Sheena]:             https://github.com/sheenaajay
   [Supported Modules]:  https://puppetlabs.github.io/iac/modules/
+  [TP]:                 https://github.com/tphoney
```
Comment (sanfrancrisko) [inline bin/blog_tools/blog_template/status-update-template.md.erb:46]: Thanks for that 😃 👍

**Review by sanfrancrisko — COMMENTED** [review]
Just one minor typo, but otherwise LGTM 👍

File: _posts/2021-01-08-status-update.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+## Happy New Year!
+
+On behalf whole IAC team we would like to take the opportunity to wish all readers a Happy New Year! We hope you all had a great Christmas. The team took the opportunity enjoy the holidays, recharge our batteries and are now ready to tackle 2021 head on.
+
+![2021]({% link /assets/2020-01-08-status-update/2021.jpg %})
+
+## Well Hello TP!
+
+TP Honey has returned to the IAC team. [TP][TP] has been with Puppet for quite a while and has worked on multiple teams. This week we see him return to the IAC team to lend some expertise. [TP][TP] specialises in all things modules and has been a leader in the development of [litmus][litmus]. We are so delighted to have him onboard with us and you will see him around a lot more. You may recognise [TP][TP] as he has done some pretty cool Puppetize talks. Check out one of my favourites called [litmus_and_pdk][Litmus and PDK].
```
Comment (DavidS) [inline _posts/2021-01-08-status-update.md:30]:
```suggestion
TP Honey has returned to the IAC team. [TP][TP] has been with Puppet for quite a while and has worked on multiple teams. This week we see him return to the IAC team to lend some expertise. [TP][TP] specialises in all things modules and has been a leader in the development of [litmus][litmus]. We are so delighted to have him onboard with us and you will see him around a lot more. You may recognise [TP][TP] as he has done some pretty cool Puppetize talks. Check out one of my favourites called [Litmus and PDK][litmus_and_pdk].
```

File: _posts/2021-01-08-status-update.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  - triage
+---
+
+## Happy New Year!
+
+On behalf whole IAC team we would like to take the opportunity to wish all readers a Happy New Year! We hope you all had a great Christmas. The team took the opportunity enjoy the holidays, recharge our batteries and are now ready to tackle 2021 head on.
+
+![2021]({% link /assets/2020-01-08-status-update/2021.jpg %})
+
+## Well Hello TP!
```
Comment (DavidS) [inline _posts/2021-01-08-status-update.md:28]: "Happy New TP" to fit in with the rest of the headers? :rofl:

File: _posts/2021-01-08-status-update.md (hunk 3 above)
Comment (DavidS) [reply _posts/2021-01-08-status-update.md:30]: seems like a few other links also are suffering from this - please do a manual build locally and review all links before merging!

Comment (pmcmaw) [reply _posts/2021-01-08-status-update.md:30]: thanks for that! Fixed them but will build locally to make sure I dont miss anything :-)

File: _posts/2021-01-08-status-update.md (hunk 4 above)
Comment (pmcmaw) [reply _posts/2021-01-08-status-update.md:28]: Niceeeee..... Good shout :-)

**Comment by Disha-maker** [comment]
Also, can you add in a following paragraph.... Check out our intern's journey during the first week of 2021 [here](https://puppetlabs.github.io/iac/docs/life_of_intern.html) 

Thank you :)

**Review by sanfrancrisko — APPROVED** [review]

**Pull request merged**